src/transformers/models/sam/modeling_sam.py

```diff
@@ -1270,6 +1270,31 @@ def forward(
         return_dict=None,
         **kwargs,
     ) -> List[Dict[str, torch.Tensor]]:
+        r"""
+        Example:
+
+        ```python
+        >>> from PIL import Image
+        >>> import requests
+        >>> from transformers import AutoModel, AutoProcessor
+
+        >>> model = AutoModel.from_pretrained("facebook/sam-vit-base")
+        >>> processor = AutoProcessor.from_pretrained("facebook/sam-vit-base")
+
+        >>> img_url = "https://huggingface.co/ybelkada/segment-anything/resolve/main/assets/car.png"
```

Comment:
The image should be moved out of a personal account into an internal one

Reply:
Awesome ok, will move it

Reply:
3863c2e